datus/tools/llms_tools/analyze_sql_history.py

```
@@ -90,7 +90,7 @@ def extract_single_summary(llm_tool: LLMTool, item: Dict[str, Any], index: int)
 
 
 def generate_classification_taxonomy(
-    llm_tool: LLMTool, items: List[Dict[str, Any]], existing_taxonomy: Dict[str, Any] = None
+    model: LLMBaseModel, items: List[Dict[str, Any]], existing_taxonomy: Dict[str, Any] = None
```

Ruff (0.14.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)